app/Nova/Actions/ExportAttendance.php

```diff
@@ -84,10 +89,16 @@ public function handle(ActionFields $fields, Collection $models): array
 
         $attendables_array = $attendables->sortKeys()->keys()->all();
 
-        Storage::append($file, 'GTID,'.implode(',', $attendables_array));
+        Storage::append($file, 'Name,Email,Major,'.implode(',', $attendables_array));
```

Comment:
Did you ask about UIDs?

Reply:
Since we have the information easily accessible now I'd prefer to just give it to them as requested.

Reply:
Ok but this is objectively a dumb way to identify a GT person

Reply:
It's consistent with what the other orgs give them though.

Reply:
Mmmmmmmk